manager/assets/modext/widgets/resource/modx.tree.resource.js

```diff
@@ -657,7 +657,7 @@ Ext.extend(MODx.tree.Resource,MODx.tree.Tree,{
         if (Ext.isObject(o)) {
             Ext.apply(types,o);
         }
-        var coreTypes = ['modDocument','modWebLink','modSymLink','modStaticResource'];
+        var coreTypes = ['MODX\\Revolution\\modDocument','modWebLink','modSymLink','modStaticResource'];
```

Comment:
Other classes also should be with namespaces.

Reply:
Thanks @alroniks , I ran out of time unfortunately. I'm aiming to get this completed today! :)

Reply:
Done!